nens_auth_client/conf.py

```diff
@@ -19,6 +19,13 @@ class NensAuthClientAppConf(AppConf):
     INVITATION_EMAIL_SUBJECT = "Invitation"
     INVITATION_EXPIRY_DAYS = 14  # change this to change the default expiry
 
+    ERROR_USER_DOES_NOT_EXIST = "No user account available for these credentials."
+    ERROR_USER_INACTIVE = "This account was set to inactive."
+    ERROR_INVITATION_DOES_NOT_EXIST = "The invitation does not exist."
+    ERROR_INVITATION_UNUSABLE = "The invitation has been used already."
+    ERROR_INVITATION_EXPIRED = "The invitation has expired."
+    ERROR_INVITATION_WRONG_USER = "This invitation was not intended for the current user."
```

Comment:
Voor nu goed genoeg.

Bij sommigen zou er later misschien een "hoe los ik het op" bij kunnen. "Mail helluppie@nens" bijvoorbeeld. Daar werd toch ook al naar gekeken?

Reply:
Ja klopt, maar dat wordt snel applicatie-specifiek dus vandaar dat ik t eerst maar eens in Lizard ga instellen.